samza-api/src/main/java/org/apache/samza/context/Context.java

```diff
@@ -76,4 +76,13 @@
    * @throws IllegalStateException if no {@link ApplicationTaskContextFactory} was provided for the application
    */
   ApplicationTaskContext getApplicationTaskContext();
+
+  /**
+   * Gets the {@link ExternalContext} that was created outside of the application.
+   * <p>
+   * Use {@link org.apache.samza.runtime.ApplicationRunner#run(ExternalContext)} to provide this context.
```

Comment:
Since this is Samza API, can import the class for readability instead of using the FQN. Same for other classes.

Reply:
I thought you hadn't pushed the change yet to avoid the unused import check. It looked like other Samza API classes still were using the FQN. Could you confirm?

Reply:
I did, although I only cleaned up javadocs for classes in High Level API (https://github.com/apache/samza/pull/760).

Reply:
Thanks for confirming. I'll change this.